### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by sabas**
Tweaking the phrase

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ Symbolic techniques include anything representable in graphics or even 3D, so le
 
 ![](img/scales.jpg)
 
-Symbolization tends to highlight two different characteristics of data: sequential and categorical. Sequential, or continuous data could also be called linear - it tends to be number values within a set range, like graduation rates between 0 and 100, or elevation. Categorical, or discrete data is, instead, one of a set number of values - like 'true', 'false', 'democrat' or 'republican'.
+Symbolization tends to highlight two different characteristics of data: sequential and categorical. Sequential, or continuous, data could also be called linear - it tends to be number values within a set range, like graduation rates between 0 and 100, or elevation. Categorical, or discrete, data is, instead, one of a set number of values - like 'true', 'false', 'democrat' or 'republican'.
```
Comment (tmcw) [inline index.md]: This is an unnecessary `,` in both spots.

File: index.md [hunk 2]
```diff
@@ -117,13 +117,13 @@ This branch of Earth sciences, called [geodesy](https://en.wikipedia.org/wiki/Ge
 
 ![](img/projections.jpg)
 
-Projections are what we call the mathematical equations that do the trick of turning the world into some flat shape that fits on a printout or a computer screen. It's a messy task to do, this transformation - there's no way to smoosh the world onto a screen without distorting it in some way. You either lose direction, or relative size, or come out with something very weird looking.
+Projections are those mathematical equations that do the trick of turning the world into some flat shape that fits on a printout or a computer screen. It's a messy task to do, this transformation - there's no way to smoosh the world onto a screen without distorting it in some way. You either lose direction, or relative size, or come out with something very weird looking.
```
Comment (tmcw) [inline index.md:120]: We could just drop this part of the sentence entirely - `Projections are mathematical equations...`

Comment (sabas) [inline index.md:120]: Okay

File: index.md (hunk 1 above)
Comment (sabas) [inline index.md]: My fault, I'm not a native speaker...

Comment (tmcw) [inline index.md]: No worries - these are pretty rough sentences that definitely need tweaking to be intelligible

**Comment by sabas** [comment]
Fixed

**Pull request closed without merging**